spec/changelog.yml

```diff
@@ -10,11 +10,13 @@
   - description: Remove deprecated `license` field.
     type: enhancement
     link: https://github.com/elastic/package-spec/issues/331
-- version: 1.16.1-next
+- version: 1.16.2-next
   changes:
   - description: Prepare for next patch release
     type: enhancement
-    link: https://github.com/elastic/package-spec/pull/398
+    link: https://github.com/elastic/package-spec/pull/
+- version: 1.16.1
```

Comment:
@jsoriano The next step is to prepare the changelog for the `1.16.1` release. Should we remove `1.16.2-next` and `2.0.0-next` sections temporarily?

Reply:
`1.16.2-next` should be removed, yes, and added in another change. `2.0.0-next` can stay.